### PR TITLE
Use const and static for member functions

### PR DESCRIPTION
**Pull request by LumioseSil**
This shows that these are member functions that do not modify a class's data.

### CODE REVIEW

File: absl/strings/string_view.cc [hunk 1]
```diff
@@ -64,7 +64,7 @@ std::ostream& operator<<(std::ostream& o, string_view piece) {
     size_t rpad = 0;
     if (static_cast<size_t>(o.width()) > piece.size()) {
       size_t pad = static_cast<size_t>(o.width()) - piece.size();
-      if ((o.flags() & o.adjustfield) == o.left) {
+      if ((o.flags() & std::ostream::adjustfield) == std::ostream::left) {
```
Comment (derekmauro) [inline absl/strings/string_view.cc]: This change doesn't seem consistent with the rest of the changes in this change. I would have to ask that we avoid the churn for this one.

**Review by derekmauro — CHANGES_REQUESTED** [review]

**Comment by AtariDreams** [comment]
Addressed! @derekmauro

File: absl/crc/internal/crc32c_inline.h [hunk 2]
```diff
@@ -28,7 +28,7 @@ namespace crc_internal {
 // CRC32C implementation optimized for small inputs.
 // Either computes crc and return true, or if there is
 // no hardware support does nothing and returns false.
-inline bool ExtendCrc32cInline(uint32_t* crc, const char* p, size_t n) {
+inline bool ExtendCrc32cInline(const uint32_t* crc, const char* p, size_t n) {
```
Comment (derekmauro) [inline absl/crc/internal/crc32c_inline.h]: Please see line 56. This change doesn't build.

**Review by derekmauro — CHANGES_REQUESTED** [review]

**Comment by LumioseSil** [comment]
Fixed! @derekmauro

**Pull request merged**